apps/docs/content/components/Heading.mdx

```diff
@@ -104,15 +107,85 @@ Use the `weight` prop to adjust the default visual appearance if required.
 </>
 ```
 
+### Width
+
+The recommended text width is automatically applied to Headings based on their position on the type scale.
+
+Use the `width` prop to override the default visual appearance if required.
+
+```jsx live
+<>
+  <Heading as="h3" width="condensed">
+    condensed
+  </Heading>
+  <Heading as="h3" width="normal">
+    normal
+  </Heading>
+  <Heading as="h3" width="wide">
```

Comment:
Am I misunderstanding the consts, or should this be `width="expanded"`? 🤔

Reply:
🤦 Yes, thanks for spotting that. Docs look much better now. 

![Screenshot 2023-02-14 at 17 16 34](https://user-images.githubusercontent.com/13340707/218810215-56910a8c-8707-4b53-97d0-de7e0365e127.png)